pom.xml

```diff
@@ -5,7 +5,7 @@
 
     <groupId>co.tryterra</groupId>
     <artifactId>terra-client</artifactId>
-    <version>0.0.3</version>
+    <version>0.0.4</version>
```

Comment:
You don't need to do this. The pipeline bumps the version automatically before releasing.

Reply:
cool